cs/v1/CS_v1.0.0.yaml

```diff
@@ -515,17 +515,17 @@ components:
         facilityTypeCode:
           description: |
             The code to identify the specific type of facility. The code indicates which role the facility plays during the transportCall.
-            - BORD (Border)
-            - CLOC (Customer Location)
-            - COFS (Container Freight Station)
-            - OFFD (Off Dock Storage)
-            - DEPO (Depot)
-            - INTE (Inland Terminal)
-            - POTE (Port Terminal)
-            - PBPL (Pilot Boarding Place)
-            - BRTH (Berth)
-            - RAMP (Ramp)
-            - WAYP (Waypoint)
+            - `BORD` (Border)
+            - `CLOC` (Customer Location)
+            - `COFS` (Container Freight Station)
+            - `OFFD` (Off Dock Storage)
+            - `DEPO` (Depot)
+            - `INTE` (Inland Terminal)
+            - `POTE` (Port Terminal)
+            - `PBPL` (Pilot Boarding Place)
+            - `BRTH` (Berth)
+            - `RAMP  (Ramp)
```

Comment:
hi, missing the closing in ramp - `RAM**P  (**Ramp) in alignment with the others. @palatsangeetha